CHANGELOG.md

```diff
@@ -34,6 +35,103 @@
+The Reports have been reorganized in a more structured way, and the implementations can now add information on how the tests have been run and provide reproduction steps.
+
+Leading Contributors: @mlavacca, @shaneutt
+
+### Port in ParentRefs
+
+You can now use a `port` field in ParentRefs to attach to Gateways, Services, or
+other parent resources.
+
+Leading Contributor: @frankbu
+
+## Experimental Channel
+
+### Session Persistence + BackendLBPolicy
+Session Persistence is being introduced to Gateway API via a new policy
+(BackendLBPolicy) for Service-level configuration and as fields within HTTPRoute and
+GRPCRoute for Route-level configuration. The BackendLBPolicy and Route-level
+APIs provide the same session persistence configuration, including session
+timeouts, session name, session type, and cookie lifetime type.
+
+Leading Contributors: @gcs278, @ginayeh
+
+### Gateway Client Cert Verification
+Gateways can now configure client cert verification for each Gateway Listener by introducing a new field
+`frontendValidation` field within `tls`. This field supports configuring a list of CA Certificates that can be used as a trust anchor to validate the certificates presented by the client.
+
+Leading Contributors: @arkodg
+
+### BackendTLSPolicy
+As part of a broader goal of making our TLS terminology more consistent
+throughout the API, we've introduced some breaking changes to BackendTLSPolicy.
+This has resulted in a new API version (v1alpha3) and will require any existing
+users of this policy to uninstall the v1alpha2 version before installing this
+newer version.
+
+Leading Contributors: @candita
+
+### Gateway Params
+Gateways now feature a new field that allows references to
+implementation-specific parameters, similar to GatewayClass.
+
+Leading Contributors: @howardjohn
+
+## Everything Else
+
+### Validation Changes
+- TLS Configuration is no longer required on Gateway Listeners to enable more
+  flexible TLS configuration. (#2721, @robscott)
+
+### Conformance
+
+- Fixed GatewayWithAttachedRoutes conformance test to not check that the
+  HTTPRoute status includes an "Accepted: False" condition because this is not
+  required by the specification. (#2548, @frankbu)
+
+### Cleanup
+
+- The validating webhook has been removed. CEL validation is now built-in to
+  CRDs and replaces the webhook. (#2595, @robscott)
+- BackendTLSPolicy WellKnownCACerts field has been updated to
+  implementation-specific support (#2741, @sunjayBhatia)
+- Clarify policy attachment by two of the same policy types when using section
+  names. (#2442, @maleck13)
+
+
```

Comment:
Am I blanking or did we have more GWCTL changes? :thinking: 

I was just thinking about this, since GWCTL is one of the more user-facing things we have... I swear I remember there being more stuff...

Reply:
@gauravkghildiyal is there anything we want to include in the changelog for gwctl?

Reply:
```suggestion
### gwctl

* We've extended the `get` command to support gateways, gatewayclasses, and
  namespaces. (#2865, #2782, #2847, @jongwooo)
* The `get` command now provides more detailed information for httproutes,
  policies, and policycrds. (#2805, #2808, #2811, @jongwooo)
* `describe` command now supports descriptions of policycrds and namespaces.
  (#2872, #2836, @Devaansh-Kumar)
* We've added the ability to filter resources using labels (through the `-l`
  flag) with both the `get` and `describe` commands. (#2892, #2915, #2934,
  @yeedove)
* Bug fix: Prevent panic when describing gatewayclasses with no description
  (#2894, @pmalek)
```